app/styles/layouts/auth.css

```diff
@@ -17,6 +17,10 @@
     margin-bottom: 1.5rem;
 }
 
+.gh-signin .form-group.error {
```

Comment:
It shouldn't be too difficult to replicate. I'll look over the signin again once i've refactored the rest of the code 😄